protocol.md

```diff
@@ -72,7 +72,8 @@ A first `device_creation` block is constructed with the *virtual device*'s publi
 
 A second `device_creation` block is created with the *physical device*'s [Device Encryption Key Pair] and [Device Signature Key Pair]. *Tanker Core* creates a new ephemeral key pair, signs this block with the ephemeral private key and uses the *virtual device*'s [Device Signature Key Pair] to sign the *delegation* and fill the block's *delegation_signature* field.
 
-Finally, both `device_creation` *block*s are pushed to the *Trustchain*, the *virtual device* before the *physical one*, and the [Verification Key] is encrypted with the [User Secret] and uploaded to the *Tanker server* according to the [Verification Method] used by the *user*.
+Finally, both `device_creation` *block*s are pushed to the *Trustchain*, the *virtual device* before the *physical one*, and the [Verification Key] is encrypted and uploaded to the *Tanker server* according to the [Verification Method] used by the *user*.
+If the [Verification Method] is an end-to-end method (e.g. the E2E Passphrase method), the [Verification Key] to be uploaded is encrypted once with the [User Encryption Key Pair] and once with the E2E method's secret value (e.g. the E2E passphrase). Otherwise, for non-E2E methods, it is encrypted only with the [User Secret].
```

Comment:
- end-to-end or non-E2E choose one

Reply:
IMO `end-to-end (E2E)` is at line 186. Use `end-to-end` before that (here and at line 111)

Reply:
Ack, switched to end to end for both